test/test-buildURL.js

```diff
@@ -223,11 +223,31 @@ describe('URL Builder:', function describeSuite() {
             assert.equal(expectation, result);
         });
 
+        it('does not modify its input-argument', function testSpec() {
+            var params = {
+                    w: 400,
+                    h: 300
+                },
+                expectation = '?w=400&h=300',
+                result = client._buildParams(params);
+
+            assert.equal(params.w, 400);
+            assert.equal(params.h, 300);
+            assert.equal(expectation, result);
```

Comment:
Likewise, these assertions don't seem to match the test case. I would recommend dropping them.

Reply:
I though the goal of this test was to ensure it does not modify its input? I mean, it's pretty trivial, but perhaps redundant?

Reply:
I cleaned this up.